playbook-website/app/javascript/site_styles/_site-style.scss

```diff
@@ -16,6 +16,10 @@ body {
   }
 }
 
+iframe body {
```

Comment:
Can we make this into a selector?

Reply:
This code was a test. It has been removed.